### DISCUSSION TITLE
RTSP vs MQTT

### DISCUSSION THREAD

**Discussion by jpbaril** [QuantumEntangledAndy/neolink#274] [Q&A] [answered]
Basic newbie question: what are the differences or reasons to choose between RTSP vs MQTT, or maybe both at the same time?
(knowing I would like to then integrate the camera into Home Assistant for live streaming on demand, so not continuius)

**Comment by QuantumEntangledAndy** [comment, marked as answer]
MQTT handles command and control

RTSP handles the video
